fern/docs-config/definition/docs.yml

```diff
@@ -7,6 +7,9 @@ types:
         docs: used as tab bar title, and in the navbar if no logo is defined
       logo: optional<LogoConfiguration>
       favicon: optional<string>
+      backgroundImage:
```

Comment:
let's make this `background-image` to match `navbar-links`

Reply:
Done!